elementtype.go

```diff
@@ -37,6 +37,7 @@ const (
 	ElementEBMLDocTypeVersion
 	ElementEBMLDocTypeReadVersion
 
+	ElementCRC32
```

Comment:
I did not attempt to implement any kind of CRC verification. Merely adding the element type allows parsing a file containing them, so I believe this is a good first step.